src/lando/headless_api/models/automation_job.py

```diff
@@ -49,6 +49,9 @@ class AutomationJob(BaseModel):
     # Text describing errors when status != LANDED.
     error = models.TextField(default="", blank=True)
 
+    # Specification of RelBranch to push changes to.
+    relbranch_specifier = models.JSONField(null=True, blank=True)
```

Comment:
Since this appears to be for only two fields, one of which is optional, why not have two separate fields rather than a JSON field? e.g. `relbranch_name` and `relbranch_commit_sha` or something similar. I think this would simplify the rest of the implementation.